WISE-Community/WISE-Client
# fix(Authoring): Fix add step before for branch path steps

## Changes
- Performing add step before on the first step in a branch path now works

Note:
- Add step before on the merge step does not work
- Add step before on first step in a branch path that is in an inactive lesson does not work

## Test
- Add step before on the first step in a branch path and make sure it works for all 4 ways you can add a step
  - Create Your Own
  - Import From Another Unit
  - Automated Assessment
  - Interactive Simulation
